app/assets/javascripts/dragdrop.js

```diff
@@ -17,44 +17,36 @@ jQuery(function() {
     );
   }
 
-  $('.dropzone').on('dragover',function(e) {
-    e.preventDefault();
```

Comment:
I stuck these listeners in the `$('.dropzone').each` further down. Having them inside that function lets me use `$(this)` to attach an eventListener... Which reduces the number of times that the same eventListener is attached to the same element.